client/src/components/ClassCard.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Card, Button } from 'semantic-ui-react';
+
+import { editUserClasses } from '../utils/apiWrapper';
+import '../css/ClassCard.scss';
+
+
+function ClassCard(props) {
+  const { classCardText } = props;
+  return (
+    <Card>
+      <Card.Content className="card-content-container">
+        <Card.Description>{ classCardText }</Card.Description>
+        <Button basic color="red" onClick={ editUserClasses() }>X</Button>
```

Comment:
Change `onClick={ editUserClasses() }` to `onClick={editUserClasses}` (i.e. remove the parentheses). Let's discuss the reasoning in person